instrumentation/lettuce-4.3/build.gradle

```diff
@@ -0,0 +1,24 @@
+dependencies {
+    implementation(project(":newrelic-weaver-api"))
+    implementation(project(":agent-bridge"))
+    implementation 'biz.paluch.redis:lettuce:4.4.1.Final'
+    testImplementation('org.testcontainers:testcontainers:1.17.1')
```

Comment:
Are we requiring Docker to run non-AIT agent tests now? If so this needs to be stated as a requirement in the agent readme.